assets/express/src/api/user/dto/user.dto.ts

```diff
@@ -0,0 +1,38 @@
+import { z } from 'zod';
```

Comment:
Having a file per route but one file for all DTOs is kinda weird to me... What about if version 2 of a route needs a change?